### PR TITLE
🌐 [i18n-KO] Translated `llama2.md` to Korean

### PR DESCRIPTION
**Pull request by mjk0618**
# What does this PR do?

Translated the `llama2.md` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/transformers/issues/20179

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [ ] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)

May you please review this PR? @sgugger, @ArthurZucker, @eunseojo

### CODE REVIEW

File: docs/source/ko/model_doc/llama2.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+- 변환 후 모델과 토크나이저는 다음과 같이 로드할 수 있습니다:
+
+```python
+from transformers import LlamaForCausalLM, LlamaTokenizer
+
+tokenizer = LlamaTokenizer.from_pretrained("/output/path")
+model = LlamaForCausalLM.from_pretrained("/output/path")
+```
+
+스크립트를 실행하려면 모델을 float16 정밀도로 전체 RAM에 호스트할 수 있을 만큼 충분한 CPU RAM이 필요합니다 (가장 큰 버전이 여러 체크포인트로 제공되더라도 각 체크포인트는 모델 가중치의 일부만을 포함하므로 모두 RAM에 로드해야 합니다). 75B 모델의 경우, 따라서 145GB의 RAM이 필요합니다.
```
Comment (jungnerd) [inline docs/source/ko/model_doc/llama2.md:61]:
```suggestion
스크립트를 실행하려면 모델을 float16 정밀도로 전부 호스트할 수 있을 만큼 충분한 CPU RAM이 필요합니다 (가장 큰 버전이 여러 체크포인트로 제공되더라도 각 체크포인트는 모델 가중치의 일부만을 포함하므로 모두 RAM에 로드해야 합니다). 75B 모델의 경우, 총 145GB의 RAM이 필요합니다.
```
문장을 조금 다듬어 보았습니다!

**Review by jungnerd — COMMENTED** [review]
좋은 번역 감사합니다! 😆
일부 리뷰를 해봤습니다. 검토 부탁드려요!

**Comment by mjk0618** [comment]
> 좋은 번역 감사합니다! 😆 일부 리뷰를 해봤습니다. 검토 부탁드려요!

리뷰 감사합니다! 말씀해주신 부분 반영하여 커밋하였습니다.

**Comment by EnginnerDog** [comment]
Uhhh hellooo? This PR should unexist.

File: docs/source/ko/_toctree.yml [hunk 2]
```diff
@@ -339,6 +339,10 @@
         title: (번역중) LED
       - local: model_doc/llama
         title: LLaMA
+      - local: model_doc/llama2
+        title: LLaMA2
+      - local: model_doc/llama
```
Comment (jungnerd) [inline docs/source/ko/_toctree.yml:344]:
```suggestion
      - local: model_doc/llama
```
위와 중복되는 라인이라 삭제가 필요합니다!

**Comment by jungnerd** [comment]
> Uhhh hellooo? This PR should unexist.

Hi! I didn't understand what you meant, why shouldn't this PR exist?

**Comment by mjk0618** [comment]
May you please review this PR? @sgugger, @ArthurZucker, @eunseojo

File: docs/source/ko/_toctree.yml [hunk 3]
```diff
@@ -339,6 +339,9 @@
         title: (번역중) LED
       - local: model_doc/llama
         title: LLaMA
+      - local: model_doc/llama2
+        title: LLaMA2
+        title: LLaMA
```
Comment (jungnerd) [inline docs/source/ko/_toctree.yml:344]:
```suggestion
```
이 라인도 삭제해야합니다!

Comment (mjk0618) [reply docs/source/ko/_toctree.yml:344]: 한꺼번에 삭제가 안됐군요..! 확인채주셔서 감사합니다. 커밋하였습니다!

**Review by ArthurZucker — APPROVED** [review]
Looks good to me! Thanks for translating this 😉

**Review by stevhliu — APPROVED** [review]
Awesome, thanks for your translation! 🤗

**Pull request merged**